openparticle/__init__.py

```diff
@@ -6,6 +6,7 @@
     AntifermionOperator,
     BosonOperator,
     NumberOperator,
```

Comment:
Should we remove `NumberOperator`?

Reply:
For sure. I didn't just because it broke some tests and I wasn't sure what (or who) else was using it, but if you wanna go ahead and delete it/fix what's broken then I'm cool with that!